src/Kount/Util/Khash.php

```diff
@@ -52,7 +52,7 @@ public static function getSaltPhrase() {
   public static function hashPaymentToken ($token) {
     $firstSix = mb_substr($token, 0, 6, 'latin1');
     $hash = self::hash($token, 14);
```

Comment:
Please, add a simple test demonstrating that trying to khash a null value would return an empty string. Thanks!